simple/reprovide.go

```diff
@@ -77,7 +77,7 @@ func (rp *Reprovider) Run() {
 
 		err := rp.Reprovide()
 		if err != nil {
-			logR.Debug(err)
+			logR.Error(err)
```

Comment:
This should include a bit more information. Something like `lotR.Errorf("failed to reprovide: %s", err)`.

This is also going to be a bit spammy when offline but that shouldn't be that big of an issue.